mocks/index.d.ts

```diff
@@ -52,11 +66,64 @@ export type SecretManagerRequestBody = {
   SecretId: string;
 };
 
-export type GetItemBody = { id: string };
 
-export type SearchQueryBody = Search_RequestBody;
+type FieldValue = boolean | undefined | number | string
```

Comment:
I can't get these types directly from opensearch for some reason, so I have copied and pasted the ones we need here. I know that's not entirely sustainable, but I wasn't sure what else to do